Purchases/Logging/Logger.swift

```diff
@@ -14,57 +14,42 @@
 
 import Foundation
 
-@objc(RCLogLevel) public enum LogLevel: Int {
+@objc(RCLogLevel) public enum LogLevel: Int, CustomStringConvertible {
```

Comment:
this is the same thing we do for other enums, particularly String files, so it seemed like a good idea to retain the pattern